torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -3260,12 +3269,12 @@ void TensorExprKernel::compile() {
     const auto& tt = output->type()->expect<TensorType>();
     auto sizes = *tt->sizes().concrete_sizes();
     tensorOutputSizes_.push_back(sizes);
-    auto strides = *tt->strides().concrete_sizes();
+    auto strides = tt->strides().concrete_sizes();
 
     // If the tensor is not dense or overlaps, we have
     // no way of matching the profiled striding
-    if (denseAndNonOverlapping(sizes, strides)) {
-      tensorOutputStrides_.push_back(*tt->strides().concrete_sizes());
```

Comment:
Just curious: how was this working before? Was this causing a bug when strides are not concrete?

Reply:
This has never been the case as the only scenario this was used is in the PE, and PE fills in both strides and sizes. In the AOT scenario we currently have a way to populate sizes, but not strides - hence the change.

Reply:
Why can't we populate strides AOT?

Reply:
We can, I just felt for some unknown reason that it would be slightly easier this way. Eventually we should have a pass to fill in strides.